### ISSUE TITLE
1.1.4 does not work on iOS 12 (library not loaded)

### ISSUE THREAD

**Issue by Jeroenbb94** [AndyQ/NFCPassportReader#104]
When running version 1.1.4 on an iOS 12 device you get the following crash message on startup:

```
dyld: Library not loaded: /System/Library/Frameworks/CryptoTokenKit.framework/CryptoTokenKit
  Referenced from: /private/var/containers/Bundle/Application/80449E41-D4CD-4FA5-936A-A37C043B0487/CMApp.app/Frameworks/NFCPassportReader.framework/NFCPassportReader
  Reason: image not found
```

Adding the CryptoTokenKit framework to the app won't help either, because the framework is iOS 13 only (https://developer.apple.com/documentation/cryptotokenkit). Could 1.1.4 be resolved to support iOS 12 again or can de pod spec be updated to minimal iOS 13?

This issue is happening from 1.1.2 (1.1.1 still works on iOS 12)

**Comment by Jeroenbb94** [comment]
The crash is happing because of the import CryptoTokenKit in Utils.swift. I'll try to change it to the following:
```
#if canImport(CryptoTokenKit)
    import CryptoTokenKit
#endif
```

**Comment by Jeroenbb94** [comment]
Above didn't resolve it, though adding `-weak_framework "CryptoTokenKit"` to the spec.xconfig in the pod spec did since it is a framework that is being used in the codebase but not specified for cocoapods.

**Comment by zaidahmed2468** [comment]
Do you have an eta when you will fix this issue (give support to iOS 12) ?

**Comment by AndyQ** [comment]
This should be fixed in 1.1.6 (due soon)
